### PR TITLE
5467 NameRequest UI Updates

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* /bcgov/entity#5467

*Description of changes:*

* Invision Designs: https://projects.invisionapp.com/d/main/#/console/20760950/436810879/comments/138149634

* Requirements documents: https://docs.google.com/spreadsheets/d/1rIv-AN59jfsHsBIvqKa9ZRFQ63O5aQaG-ol5HWLYxZ0/edit#gid=561821440

- [x] Add Beta Tag to title
- [x]  New Name Search & Existing Name Request - Font size/spacing etc.
- [x]  'I need a name to:' - font size/weight.
- [x]  Move 'Check to see if you need to file a Name Request' link.
- [x]  Verify no text clipping in all fields.
- [x]  'Name Search Field' - update verbiage and icon colour.
- [x] CheckBoxes - Update Text and no default selection.
- [x] ** Re-work of 'English' checkbox functionality.  ** 
- [x] Implement 'SEARCH NAME' btn 
- [x] Updated Checkbox tool tip style and alignment
------------------------------------------------------------
- [x] Convert first option overlay to Dropdown.
- [x] Blue hover on list items.
- [x] Update tooltip styling in all Dropdowns.
- [x] Dropdown width to fit longest option.
- [x] Tooltip hover on base dropdown list item IF truncated.
- [x] Separator line in business type Dropdown.
- [x] **Follow each FLOW to update verbiage and ensure proper options available.** ONGOING
https://docs.google.com/spreadsheets/d/1rIv-AN59jfsHsBIvqKa9ZRFQ63O5aQaG-ol5HWLYxZ0/edit#gid=1055072337
- [x] Update Input text fields labels and styling.
~- [ ] Expand unit testing.~ Will be follow up PR for sanity sake

*Notes:* 
* The backend work required is for namex to support Xpro name analysis on `Move` actions for extra provincial locations. Currently was only supported for locations of 'BC' but the requirements have been changed, so this request will currently fail until that BE task is completed.

* Said work is captured here: https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5705

* Some entities options will still need to be added/removed from some Request types. This is work captured in ...
https://app.zenhub.com/workspaces/entity-5bf2f2164b5806bc2bf60531/issues/bcgov/entity/5586

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/common/names-capture.vue [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <v-container fluid class="pa-0" id="send-to-examination-container">
       <template v-if="editMode">
         <v-row justify="end" class="mt-n3 mb-n5">
-          <v-col cols="6" class="copy-normal">I need a name to:</v-col>
+          <v-col cols="6" class="copy-normal copy-bold">I need a name to:</v-col>
```
Comment (severinbeauvais) [inline client/src/components/common/names-capture.vue:6]: Here's hoping this project moves to the existing Vuetify helper classes in the near future.
https://vuetifyjs.com/en/styles/text-and-typography/#font-emphasis etc

File: client/src/App.vue [hunk 2]
```diff
@@ -230,8 +230,6 @@ export default class App extends Mixins(SessionTimerMixin) {
 #main-column
   padding: 0
   margin: 0 auto 0 auto
-  width: 1380px
```
Comment (severinbeauvais) [inline client/src/App.vue:233]: Are there any responsive breakpoints to update?

File: client/src/sass/main.sass [hunk 3]
```diff
@@ -117,17 +117,18 @@
 
 .h5, h5
   color: $dk-text
-  font-size: 16px
+  font-size: 18px
```
Comment (severinbeauvais) [inline client/src/sass/main.sass:120]: Here's hoping this project moves to relative units (ie, rem) to facilitate sizing/scaling in the near future.

https://www.google.com/search?q=why+use+rem+px

TLDR;
https://css-tricks.com/is-it-better-to-use-ems-rems-than-px-for-font-size/

File: client/src/App.vue [hunk 4]
```diff
@@ -230,8 +230,6 @@ export default class App extends Mixins(SessionTimerMixin) {
 #main-column
```
Comment (severinbeauvais) [inline client/src/App.vue:230]: I can't add a comment on the favicon.ico file so I'll do it here.

Please use a favicon PNG file from our UI projects as it has a transparent background.

File: client/src/components/modals/pick-entity-or-conversion.vue [hunk 5]
```diff
@@ -1,58 +1,55 @@
...
                   <template v-slot:activator="scope">
-                    <button v-on="scope.on" class="link-sm-sans-ul text-left">{{ conversion.desc }}</button>
+                    <button v-on="scope.on" class="link-sm-sans-ul entity-link">{{ conversion.desc }}</button>
                   </template>
-                  <ul class="ma-0 pa-0">
-                    <li class="no-bullet"
-                        v-html="conversion.blurb"
-                        :key="i + '-blurb'" />
-                  </ul>
+                  <div v-for="(text, i) in conversion.blurb" :key="i + '-blurb'">
```
Comment (severinbeauvais) [inline client/src/components/modals/pick-entity-or-conversion.vue:26]: I'm not sure whether keys that start with a number work...?

Can you just use `:key:"i"` here?

File: client/src/components/modals/pick-entity-or-conversion.vue [hunk 6]
```diff
@@ -1,58 +1,55 @@
...
+                  </div>
                 </v-tooltip>
               </v-col>
             </v-row>
           </v-container>
         </v-card-text>
       </template>
       <template v-else-if="!showSocietiesInfo">
-        <v-card-text class="py-4" style="display: flex; justify-items: center; width: 100%">
+        <v-card-text style="display: flex; justify-items: center; width: 100%">
```
Comment (severinbeauvais) [inline client/src/components/modals/pick-entity-or-conversion.vue:36]: Don't change but FYI: there are Vuetify helper classes for this already.

https://vuetifyjs.com/en/styles/flex/ etc

File: client/src/components/modals/pick-entity-or-conversion.vue [hunk 7]
```diff
@@ -113,6 +105,9 @@ export default class PickEntityOrConversion extends Vue {
   get location () {
     return newReqModule.location
   }
+  get locationText () {
+    return newReqModule.locationText === 'BC' ? 'British Columbia' : newReqModule.locationText
```
Comment (severinbeauvais) [inline client/src/components/modals/pick-entity-or-conversion.vue:109]: Perfect.
Whenever practical, don't have logic in the template.

File: client/src/components/new-request/search.vue [hunk 8]
```diff
@@ -119,17 +190,26 @@ export default class NewSearch extends Vue {
...
-
+  private request_action_enum = [
+    'NEW',
+    'MVE',
+    'REH',
+    'AML',
+    'CHG'
+  ]
+  private get isScreenLg () {
+    return this.$vuetify.breakpoint.width > 1440
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:211]: Why not use CSS breakpoints?

https://www.w3schools.com/css/css_rwd_mediaqueries.asp and others.

Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:211]: Or Vuetify breakpoints?

https://vuetifyjs.com/en/features/breakpoints/

File: client/src/components/new-request/search.vue [hunk 9]
```diff
@@ -190,44 +273,59 @@ export default class NewSearch extends Vue {
   get locationOptions () {
     return newReqModule.locationOptions
   }
+  get locationText () {
+    return newReqModule.locationText
+  }
   get nameIsEnglish () {
-    return newReqModule.nameIsEnglish
+    return !newReqModule.nameIsEnglish
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:280]: This doesn't make sense 😆

File: client/src/components/new-request/stats.vue [hunk 10]
```diff
@@ -1,16 +1,17 @@
...
-          <div class="stats-content-inner-1 text-center">
-            <div class="stats-value h3-lt">{{ autoApprovedCount }}</div>
-            <div class="stats-unit">Names</div>
-          </div>
-          <div class="stats-content-inner-2">
-            <u>Auto-Approved</u> Today.<br>
-            No Wait Time
-          </div>
-        </div>
+<!--      COMMENTED OUT FOR FUTURE IMPLEMENTATION-->
```
Comment (severinbeauvais) [inline client/src/components/new-request/stats.vue:4]: Are you using VS Code? What do you get if you select this block and press CTRL+/ ?

File: client/src/components/tabs.vue [hunk 11]
```diff
@@ -59,6 +60,13 @@ export default class Tabs extends Vue {
 </script>
 
 <style lang="sass" scoped>
+#tabs-landing-comp
+  max-width: 1264px
+
+.tab-items
+  border-bottom-left-radius: 6px !important
+  border-bottom-right-radius: 6px !important
```
Comment (severinbeauvais) [inline client/src/components/tabs.vue:68]: https://vuetifyjs.com/en/styles/border-radius/ ?

File: client/src/models.ts [hunk 12]
```diff
@@ -54,7 +54,7 @@ export interface ConsentConflictI {
   [propName: string]: any
 }
 export interface ConversionTypesI {
-  blurb?: string
+  blurb?: string[]
```
Comment (severinbeauvais) [inline client/src/models.ts:57]: Should rename to `blurbs`.

Also below.

File: client/src/sass/overrides.sass [hunk 13]
```diff
@@ -102,19 +102,58 @@
...
+  background-color: RGBA(73,80,87,0.95) !important
+  color: #ffffff !important
   border-radius: 4px
-  box-shadow: 0 0 7px -2px $grey-5
   font-size: 12px !important
+  line-height: 18px !important
   padding: 20px !important
+  letter-spacing: 0
+
+.v-tooltip__content:after
```
Comment (severinbeauvais) [inline client/src/sass/overrides.sass:118]: Why can't v-tooltip props be used to place the tooltip at right/left/top/bottom, possibly with offsets or nudges?

https://vuetifyjs.com/en/api/v-tooltip/

File: client/src/views/landing.vue [hunk 14]
```diff
@@ -74,7 +74,7 @@ export default class Landing extends Vue {
 #landing-container
   margin: 0
   padding: 0
-  min-width: 1380px !important
+  min-width: 1080px !important
```
Comment (severinbeauvais) [inline client/src/views/landing.vue:77]: Will this be responsive on a smaller screen?

File: client/src/store/new-request-module.ts [hunk 15]
```diff
@@ -496,60 +547,64 @@ export class NewRequestModule extends VuexModule {
...
       shortDesc: 'New Request',
       value: 'NEW',
-      blurb: `Start a new business in BC. This applies to starting fresh from here or having a business in another 
-              province or country that you want to operate in BC as well.`
+      blurb: `Create a new business in British Columbia or register a business you formed in another province or 
+              territory, country or federal jurisdiction so that you may also conduct business here in BC.`
     },
     {
-      text: 'Move your Business to BC',
+      text: 'Move (continue) into BC a',
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:557]: Incomplete text?

File: client/src/store/new-request-module.ts [hunk 16]
```diff
@@ -496,60 +547,64 @@ export class NewRequestModule extends VuexModule {
...
       value: 'CNV',
-      blurb: `Convert from one business structure to another. Such as converting from a ULC to a BC Corp. You will need 
-              to identify your business with your Corp. #/Firm # (assigned by Registries).`
+      blurb: `You want to alter from one type of corporation to another.  For example you are a limited company and 
+              want to become an unlimited liability company.   You will need the incorporation number assigned to you 
+              by BC Registries and Online Services.`
     },
     {
-      text: 'Restore Using a Historical Name',
+      text: 'Restore or Reinstate a',
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:593]: For consistency, should "Reinstate" be un-capitalized?

File: client/src/store/new-request-module.ts [hunk 17]
```diff
@@ -816,16 +874,44 @@ export class NewRequestModule extends VuexModule {
...
       { text: 'BC', value: 'BC' },
-      { text: 'Canada', value: 'CA' },
-      { text: 'Foreign', value: 'IN' },
-      { text: 'Help', value: 'INFO' }
+      {
+        text: 'Extra provincial (Canada based)',
+        value: 'CA',
+        blurb: [
+          `Your existing business is currently located in any Province or Territory other than BC.`,
+          `Your existing business is currently located in any Province or Territory other than BC.`,
```
Comment (severinbeauvais) [inline client/src/store/new-request-module.ts:882]: Duplicate blurb?

**Review by severinbeauvais — COMMENTED** [review]
Some comment but looks good so far. I'm really looking forward to seeing the changes deployed!

File: client/src/components/common/names-capture.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/common/names-capture.vue:6]: Absolutely, i tried to play it safe in this ticket. The scope was already large :)

File: client/src/App.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/App.vue:233]: Not that i could see..

File: client/src/sass/main.sass (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/sass/main.sass:120]: Very cool, i like it. 
How does this work as per designs, where they are specified as specific px's. Is proper scaling between screen sizes just assumed and designs are based on the most standard screen sizes?

File: client/src/App.vue (hunk 4 above)
Comment (cameron-freshworks) [reply client/src/App.vue:230]: Good call will do, i thought it looked off, thanks!

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 5 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:26]: We can for sure. Normally i'd be worried about duplicate keys but these blurbs are tool tips.

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 6 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:36]: For sure, this was existing styling relocated, but a good idea to update where possible 👍

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 7 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:109]: Definitely!

File: client/src/components/new-request/search.vue (hunk 8 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:211]: Yea, the idea WAS to use Vuetify breakpoints here but i complicated it by comparing it to a hard value.
Should be more like `return this.$vuetify.breakpoint.lg`.

File: client/src/components/new-request/search.vue (hunk 9 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:280]: I should refactor the name 😆 
The verbiage update here called for the opposite of what it currently was, functionally as well. This was the easy path, but i'll clear this up.

File: client/src/components/new-request/stats.vue (hunk 10 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/stats.vue:4]: I use jetbrains (webstorm) in this case.
I don't get anything actually haha. My keybind is WIN + / to commented and uncomment blocks.

File: client/src/components/tabs.vue (hunk 11 above)
Comment (cameron-freshworks) [reply client/src/components/tabs.vue:68]: Cool! They have a class for everything. Will update this!

File: client/src/models.ts (hunk 12 above)
Comment (cameron-freshworks) [reply client/src/models.ts:57]: will do 👍

File: client/src/sass/overrides.sass (hunk 13 above)
Comment (cameron-freshworks) [reply client/src/sass/overrides.sass:118]: I do use those for general alignment, though the positioning isn't quite what the design calls for. This also applies the styling for the pointers.

File: client/src/views/landing.vue (hunk 14 above)
Comment (cameron-freshworks) [reply client/src/views/landing.vue:77]: Slightly, working towards that. This was the point when the main container started to break, for me at least.

File: client/src/store/new-request-module.ts (hunk 15 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:557]: i'm glad you commented on it though, it looks like i might have an extra space after the `(continue)` or maybe not. I'll check!

File: client/src/store/new-request-module.ts (hunk 16 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:593]: The documentation calls for capitalization here but will follow up since the trend has definitely been away from that pattern.

File: client/src/store/new-request-module.ts (hunk 17 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:882]: Yes and no 😛 
Each index is for a specific scenario.. some scenarios share the same description. It was easier to just assign the blurb twice where required opposed to write some conditional logic to to apply the same description twice under xxx circumstances.

File: client/src/components/common/names-capture.vue (hunk 1 above)
Comment (severinbeauvais) [reply client/src/components/common/names-capture.vue:6]: How about this? If you're in there again, tweak it, otherwise it will a future, global effort.

File: client/src/sass/main.sass (hunk 3 above)
Comment (severinbeauvais) [reply client/src/sass/main.sass:120]: It means we follow the designs but mostly convert to rems (except, say, small offset or the like).
12 px &rarr; 0.75rem
14 px &rarr; 0.875rem
16 px &rarr; 1rem
etc
And if possible, we use a few variations as possible.

Again, if you're in there, tweak it, otherwise it's future work.

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 5 above)
Comment (severinbeauvais) [reply client/src/components/modals/pick-entity-or-conversion.vue:26]: Compromise: `blurb-x` ?

(It's the leading number that I'm uncomfortable with.)

File: client/src/components/new-request/search.vue (hunk 8 above)
Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:211]: > `return this.$vuetify.breakpoint.lg`

I just learned something. That would be perfect here. Cool!

File: client/src/components/new-request/search.vue (hunk 9 above)
Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:280]: Gotcha. Thanks, please rename, lol.

File: client/src/components/common/names-capture.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/common/names-capture.vue:6]: Absolutely, good plan!

File: client/src/sass/main.sass (hunk 3 above)
Comment (cameron-freshworks) [reply client/src/sass/main.sass:120]: Definitely, this is relatively a quick adjustment in the base sass. Will do!

File: client/src/components/new-request/stats.vue (hunk 10 above)
Comment (severinbeauvais) [reply client/src/components/new-request/stats.vue:4]: I was surprised to see every line as its own comment. VS Code just adds a start and end tag for the whole block.

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 5 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:26]: Definitely agree, 100% will do 👍

File: client/src/components/new-request/search.vue (hunk 8 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:211]: Me as well! Interesting that it is a boolean eh

File: client/src/components/tabs.vue (hunk 11 above)
Comment (severinbeauvais) [reply client/src/components/tabs.vue:68]: I was going to say, "the classes are already there; let's use them".

But I now see that main.ts does NOT import the Vuetify CSS. Maybe it should? Future??? Eg,
```
import 'vuetify/dist/vuetify.min.css'
```

File: client/src/components/new-request/stats.vue (hunk 10 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/stats.vue:4]: ohhhh i see. Yea, i think i could do that as well. Maybe that is a preference i could change in the editor.

File: client/src/store/new-request-module.ts (hunk 17 above)
Comment (severinbeauvais) [reply client/src/store/new-request-module.ts:882]: OK, thanks for explaining. Short comment in there maybe?

File: client/src/sass/main.sass (hunk 3 above)
Comment (severinbeauvais) [reply client/src/sass/main.sass:120]: What we _should_ do (in the other UIs as well), is make greater use of CSS variables, for sizes, etc.

Thanks for humouring me :)  Don't take on too much extra work atm.

File: client/src/components/new-request/search.vue (hunk 8 above)
Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:211]: 🤓

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 6 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:36]: Updated!

File: client/src/components/common/names-capture.vue (hunk 1 above)
Comment (scottrumsby) [inline client/src/components/common/names-capture.vue:6]: very minor thing, but the `.copy-bold` class already uses `@extend .copy-normal` so  I don't think we need to put both classes on the same element

File: client/src/components/new-request/search.vue (hunk 9 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:280]: Found an alternative to renaming anything at all!

Screen Shot 2020-11-19 at 10 33 01 AM

Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:280]: Essentially just flip flops the state vs value, so the property name and all instances of it still make sense.

Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:280]: Yikes, but OK.

File: client/src/components/tabs.vue (hunk 11 above)
Comment (cameron-freshworks) [reply client/src/components/tabs.vue:68]: Noted! indeed it appears with these helpers we might need to import the CSS.

File: client/src/components/new-request/search.vue (hunk 9 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:280]: hahah it's not that scary!

File: client/src/components/common/names-capture.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/common/names-capture.vue:6]: ohh right! I see, good catch. Will update. copy-normal isn't needed

File: client/src/store/new-request-module.ts (hunk 17 above)
Comment (cameron-freshworks) [reply client/src/store/new-request-module.ts:882]: Will do!

File: client/src/models.ts (hunk 12 above)
Comment (cameron-freshworks) [reply client/src/models.ts:57]: Updated in next commit. A LOT of `blurb` converted to `blurbs`

**Review by scottrumsby — APPROVED** [review]
Looks fine to me at this point (assuming the other things mentioned are or will be fixed)

File: client/src/components/common/names-capture.vue [hunk 18]
```diff
@@ -3,7 +3,7 @@
     <v-container fluid class="pa-0" id="send-to-examination-container">
       <template v-if="editMode">
         <v-row justify="end" class="mt-n3 mb-n5">
-          <v-col cols="6" class="copy-normal">I need a name to:</v-col>
+          <v-col cols="6" class="font-weight-bold">I need a name to:</v-col>
```
Comment (severinbeauvais) [inline client/src/components/common/names-capture.vue:6]: Now you've lost the "copy-normal" styles. Is that OK?

File: client/src/components/new-request/search.vue [hunk 19]
```diff
@@ -1,89 +1,161 @@
 <template>
   <v-container fluid id="new-request-container" class="copy-normal">
-    <v-row justify="end">
-      <v-col cols="12" class="copy-normal">I need a name to:</v-col>
+    <v-row cols="12">
+      <v-col cols="6" class="font-weight-bold">I need a name to:</v-col>
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:4]: Do you need "copy-normal" style here as well?

File: client/src/components/common/names-capture.vue (hunk 18 above)
Comment (cameron-freshworks) [reply client/src/components/common/names-capture.vue:6]: You bet, it is inherited here from container.

File: client/src/models.ts [hunk 20]
```diff
@@ -84,7 +84,8 @@ export interface DraftReqI {
   request_type_cd?: string
 }
 export interface EntityI {
-  blurb?: string[] | string
+  blurbs?: string[] | string
```
Comment (severinbeauvais) [inline client/src/models.ts:87]: This one should be `blurb(s)?: string[] | string`.





(Kidding, but really how do you tell if is a string or an array?)

File: client/src/components/new-request/search.vue (hunk 19 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:4]: Yessir, should be all good as it is inherited.

File: client/src/models.ts (hunk 20 above)
Comment (cameron-freshworks) [reply client/src/models.ts:87]: Good question, i think with the recent refactors there isn't any instances of it being a single string, but i will confirm. 
I think i converted them all to arrays since we will have variations of the blurbs based on all the different flows 😓 
Will give this a look when i finish up the copy.

Comment (severinbeauvais) [reply client/src/models.ts:87]: I saw one structure where "blurbs" is a single string, but it might be hard to untangle where it's used as an array or a string so I'm OK with leaving it alone now.

File: client/src/components/modals/pick-entity-or-conversion.vue [hunk 21]
```diff
@@ -138,7 +133,7 @@ export default class PickEntityOrConversion extends Vue {
     }
     let cols = this.tableData.length
     // 210 per column with a max of 900px
-    return `${210 * cols > 900 ? 900 : 210 * cols}px`
+    return `${210 * cols > 900 ? 960 : 210 * cols}px`
```
Comment (severinbeauvais) [inline client/src/components/modals/pick-entity-or-conversion.vue:136]: Should this be
`if X is greater than Y : Y : X`

(iow, first check should be if > 960) ?

File: client/src/components/new-request/search.vue [hunk 22]
```diff
@@ -1,89 +1,163 @@
...
+                    v-slot:label v-on="on">
+              <template>
+                <span v-on="on">Name is a person's name</span>
+              </template>
+            </v-checkbox>
+          </template>
+          <p>Check this box if you are...</p>
           <ul>
             <li>Incorporating under your own name (eg. DR. JOE SMITH INC.)</li>
             <li>The name contains one or more names. (eg. BLAKE, CHAN & DOUGLAS INC.)</li>
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:125]: Generally `&amp;` should be used instead of `&` but if works, leave it.

File: client/src/components/new-request/search.vue [hunk 23]
```diff
@@ -1,89 +1,163 @@
...
+                    v-slot:label v-on="on">
+              <template>
+                <span v-on="on">Name contains no English words</span>
+              </template>
+            </v-checkbox>
+          </template>
+          <p>This refers to the language of the words in your name.</p>
+          <ul>
+            <li>Leave this box checked if your name contains <b>only</b> English <b>or a mix</b> of English and
+              another Language
```
Comment (severinbeauvais) [inline client/src/components/new-request/search.vue:147]: "Language" should not be capitalized (IMHO).

Leave it if design has it this way.

**Review by severinbeauvais — APPROVED** [review]

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 21 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:139]: Going to have another look at this, i think we might need set a larger minimum. Some dialogs with only 1-2 cells are pretty tight. Stay tuned!

File: client/src/components/new-request/search.vue (hunk 22 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:125]: It does seem to work. I wonder why, if it's not correct tho. Will look into it!

File: client/src/components/new-request/search.vue (hunk 23 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:147]: ah good catch, will fix!

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 21 above)
Comment (severinbeauvais) [reply client/src/components/modals/pick-entity-or-conversion.vue:139]: While you're there, it would be great to use existing breakpoints (this.$breakpoint)!

File: client/src/components/new-request/search.vue (hunk 22 above)
Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:125]: https://mrcoles.com/blog/how-use-amersands-html-encode/
https://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp

Comment (severinbeauvais) [reply client/src/components/new-request/search.vue:125]: "should" not "must" FWIW

Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:125]: Updated!

File: client/src/components/new-request/search.vue (hunk 23 above)
Comment (cameron-freshworks) [reply client/src/components/new-request/search.vue:147]: fixed!

File: client/src/components/modals/pick-entity-or-conversion.vue (hunk 21 above)
Comment (cameron-freshworks) [reply client/src/components/modals/pick-entity-or-conversion.vue:139]: updated!

**Review by severinbeauvais — APPROVED** [review]
Looks great! You are good to go.
